### PR TITLE
support projects in blob commands

### LINKED ISSUE #2954: Support project in platform buckets
Part of https://github.com/neuro-inc/no-code/issues/77
We want to be able to parameterize the project name for buckets, their content, and credentials.

### PR DESCRIPTION
**Pull request by zubenkoivan**
closes #2954

### CODE REVIEW

File: neuro-cli/src/neuro_cli/click_types.py [hunk 1]
```diff
@@ -1226,3 +1248,4 @@ async def async_shell_complete(
 def setup_shell_completion() -> None:
     add_completion_class(NewZshComplete)
     add_completion_class(NewBashComplete)
+    add_completion_class(NewBashComplete)
```
Comment (dalazx) [inline neuro-cli/src/neuro_cli/click_types.py:1251]: probably a duplicate line.

**Review by dalazx — APPROVED** [review]
one minor comment. lgtm.

File: neuro-cli/src/neuro_cli/blob_storage.py [hunk 2]
```diff
@@ -140,6 +164,7 @@ async def mkbucket(
     name: Optional[str] = None,
     cluster: Optional[str] = None,
     org: Optional[str] = None,
+    project: Optional[str] = None,
```
Comment (YevheniiSemendiak) [inline neuro-cli/src/neuro_cli/blob_storage.py:167]: minor: do we forbid the creation of buckets with the same name in different projects?
currently, I cannot do it and it goes from the API.

**Review by YevheniiSemendiak — APPROVED** [review]
Great job! Thank you :muscle:

File: neuro-cli/src/neuro_cli/blob_storage.py (hunk 2 above)
Comment (zubenkoivan) [reply neuro-cli/src/neuro_cli/blob_storage.py:167]: Should be possible. When did you test this last time?

Comment (zubenkoivan) [reply neuro-cli/src/neuro_cli/blob_storage.py:167]: I think I found a bug in buckets API, will work on this, thank you

**Pull request merged**